db/model/botData.js

```diff
@@ -111,6 +124,23 @@ module.exports = function botData(seq, dataTypes) {
           .catch((err) => reject(err))
         );
       }, // hooks.beforeCreate
+
+      afterCreate: (instance) => {
+        const changedKeys = Object.keys(instance._changed);
+        const ignoreAttributes = ['isDeleted'];
+        return realTime.publishObject(instance.toJSON(),
+          botDataEventNames.add, changedKeys, ignoreAttributes, pubOpts);
+      },
+
+      afterUpdate(instance /* , opts */) {
+        return realTime.publishObject(instance.toJSON(),
+          botDataEventNames.upd, null, null, pubOpts);
```

Comment:
same question as the other pr about changed keys and ignore attributes in afterUpdate and afterDelete

Reply:
Fix this change